### DISCUSSION TITLE
Next Redash release date?

### DISCUSSION THREAD

**Discussion by M-arghandiwal** [getredash/redash#6720] [General]
When is the new release coming out? any specific date?

**Comment by conrado** [comment]
Python 3.7 has been EOL for a bit now 🥲

**Reply by rodrigost23** [reply]
In addition to that, I need updated boto3 to use IMDSv2 as a security requirement on AWS, to avoid using key/secret...

**Reply by eradman** [reply]
> Python 3.7 has been EOL for a bit now 🥲

The update to Python 3.10 was made in https://github.com/getredash/redash/pull/6991

**Reply by rodrigost23** [reply]
Yes, and there hasn't been a stable release ever since.
